frame/App/Body/Main/Layout/components/AddItem.js

```diff
@@ -18,13 +19,17 @@ const AddItem = ({
   const { firestore } = useFirebase();
 
   const [value, setValue] = React.useState('');
+  const [priority, setPriority] = React.useState(false);
+
+  const itemPriority = priority;
 
   const addNewTodoItem = () => {
     const newItem = {
       name: value,
       description: '',
       completed: false,
       day: day ?? new Date().getDay(),
+      priority: itemPriority,
```

Comment:
Don't need to make a new var just for this
```suggestion
      priority,
```